cli/pkg/locks/lock.go

```diff
@@ -88,6 +96,29 @@ func (e *EnvironmentLockParameters) FillForm() (*HttpFormDataInfo, error) {
 	}, nil
 }
 
+func (e *AppLockParameters) GetPath() string {
```

Comment:
I was scared for a second that we are doing something on the git repo.
Maybe rename to `GetRestPath` or `GetUrlPath`?